build-logic/build.gradle.kts

```diff
@@ -1,7 +1,8 @@
 plugins {
```

Comment:
So if we've removed this plugin, how is it that the `*.kts` files are still working?

Reply:
Is the idea that `*.kts` support is intrinsic, and the plugin is required just to extend it?